packages/world/src/requireValidNamespace.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.21;
+
+import { Bytes } from "@latticexyz/store/src/Bytes.sol";
+
+import { ResourceId, WorldResourceIdInstance, WorldResourceIdLib } from "./WorldResourceId.sol";
+import { IWorldErrors } from "./IWorldErrors.sol";
+
+using WorldResourceIdInstance for ResourceId;
+
+/**
+ * @notice Checks if a given `resourceId` is a valid namespace.
```

Comment:
`requireNamespace` and `requireValidNamespace` have the same `notice` comment, maybe they could be more specific?

Reply:
whoops, that was a copy pasta issue on my part